diracx-client/src/diracx/client/_patch.py

```diff
@@ -242,6 +246,17 @@ def get_token(location: Path, token: AccessToken | None) -> AccessToken | None:
     return token
 
 
+def is_refresh_token_valid(refresh_token: str) -> bool:
+    """Check if the refresh token is still valid."""
+    # Decode the refresh token
+    refresh_payload = jwt.decode(refresh_token, options={"verify_signature": False})
+    if not refresh_payload or "exp" not in refresh_payload:
+        return False
+
+    # Check the expiration time
+    return refresh_payload["exp"] > datetime.utcnow().timestamp()
```

Comment:
`utcnow` is deprecated https://docs.python.org/3.11/library/datetime.html#datetime.datetime.utcnow